pages/components/Header.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const Header = () => {
+  return (
+    <header>
+        <nav>
+            <ul>
+                <li>
+                    <a href='#'>Home</a>
```

Comment:
Great, that you implemented a placeholder for the href so it already has the attributes of an a tag even so it does not link to a page yet.